packages/dmn-js-decision-table/src/features/annotations/components/AnnotationCell.js

```diff
@@ -12,7 +12,7 @@ export default function AnnotationCell(props) {
   } = row.businessObject;
 
   return (
-    <Cell className="annotation" elementId={ id + '__annotation' }>
```

Comment:
Why we had this suffix in the first place?

Reply:
![Kapture 2020-08-07 at 8 36 28](https://user-images.githubusercontent.com/9433996/89616714-2a4c8b00-d889-11ea-8f39-e3d1b256204d.gif)

Seems like this is causing unwanted side effects. The annotation content will be displayed as description for some reason.

Reply:
> Why we had this suffix in the first place?

That's a good question. It does not lead to any element of XML and is not used anywhere.

Reply:
Fixed :)